src/icons/ValoraV.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
```

Comment:
is this the same as the home tab's icon? the svg looks different but the icon looks the same just with different sizes, should we have a single V icon? cc @MuckT